### PR TITLE
asadiqbal08/relabeling the fields and default values

### PR DESCRIPTION
**Pull request by asadiqbal08**
#### What are the relevant tickets?
#1101 

#### What's this PR do?
Re labeling the fields with default ones.

#### Screenshots (if appropriate)
screencapture-bc-odl-local-8099-cms-pages-add-cms-certificatepage-11-2020-12-18-13_03_17

### CODE REVIEW

File: cms/models.py [hunk 1]
```diff
@@ -853,17 +853,25 @@ class CertificatePage(BootcampRunChildPage):
     template = "certificate_page.html"
     parent_page_types = ["BootcampRunPage"]
 
-    sub_title = models.CharField(
+    bootcamp_run_name = models.CharField(
         max_length=250,
         null=False,
         blank=False,
-        help_text="Specify the bootcamp certificate sub title.",
+        default="MIT Innovation and Entrepreneurship Bootcamp",
```
Comment (HamzaIbnFarooq) [inline cms/models.py:860]: I don't think we need any default value here.

**Review by HamzaIbnFarooq — COMMENTED** [review]
small change

File: cms/models.py (hunk 1 above)
Comment (asadiqbal08) [reply cms/models.py:860]: good catch, I planned to add in help_text.

**Review by HamzaIbnFarooq — APPROVED** [review]
Looks good to me.

**Comment by asadiqbal08** [comment]
@pdpinch looks good to you ?

**Review by pdpinch — APPROVED** [review]

**Pull request merged**